skrf/media/media.py

```diff
@@ -838,6 +838,8 @@ def line(self, d: NumberLike, unit: str = 'deg',
         s_def = kwargs.pop('s_def', S_DEF_DEFAULT)
         # Need to use either traveling or pseudo definition here
         # for the network to match traveling waves.
+        # the definition of the line trough a match and a delay would not
+        # works for complex characteristic impedances otherwise.
```

Comment:
typo: works.

A better explanation would be that power wave reflection coefficient has conjugation. A line with characteristic impedance equal to the port impedance does not have zero reflection coefficient even with a zero length line if port impedance is complex.

Reply:
I will rephrase in this sense.
While this is outside the scope of this PR: isn't the `media.match` method with `npy.zeros()` wrong for the power waves ?

Reply:
Done